modules/db/src/main/scala/gem/dao/UserDao.scala

```diff
@@ -33,6 +33,7 @@ object UserDao {
              g.program_role
         FROM gem_user u LEFT OUTER JOIN gem_user_program g ON g.user_id = u.id
         WHERE id = $id
+        AND "md5" = md5($pass)
```

Comment:
Yeah that's a good idea. I just wanted something simple for development. We can swap out the methods on the `Service` object to do something else.